asset-transfer-basic/rest-api-typescript/src/config.ts

```diff
@@ -283,3 +283,12 @@ export const org2ApiKey = env
   .required()
   .example('456')
   .asString();
+
+/** Enable CORS, set to be the wildcard origin
+ * THis might not be applicable for all production use cases
+ */
+export const devCorsEnabled = env
```

Comment:
Rather than add a specific CORS option (with associated tests 👀) without any origin configuration, I think I would prefer to make CORS dev mode only to reinforce that it's really not intended for production